Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:18.12.0-alpine as builder
+WORKDIR /app
+COPY package.json .
+COPY package-lock.json .
+RUN npm i
+COPY . .
+RUN npm run build
+
+
+FROM nginx:1.19.0

```

it's always a good idea to use the latest stable nginx version
```suggestion
FROM nginx:1.22
```